src/main/java/com/example/music/backend/verification/domain/VerificationTokenUtil.java

```diff
@@ -12,6 +12,6 @@ public static Date calculateExpiryDate() {
         Calendar cal = Calendar.getInstance();
         cal.setTime(new Timestamp(cal.getTime().getTime()));
```

Comment:
Why do you need this line?

Reply:
she is not needed, did stupidly, deleted